README.md

```diff
@@ -64,11 +64,47 @@ At the core of n2y are a set of python classes that represent the various parts
 
 | Notion Object Type | Description |
 | --- | --- |
+| Block | A bit of content within a Page |
+| ListItemBlock | Content that is a list (numbered, bulleted, etc.) |
+| ChildPageBlock | A Notion page that is the child of another page |
+| EquationBlock | Notion content that represents a mathmatical equation |
+| ParagraphBlock | Text content in the form of a paragraph |
+| BulletedListItemBlock | Text content in the form of a bulleted list |
+| ToDoListItemBlock | Text content in the form of a todo list |
+| NumberedListItemBlock | Text content in the form of a numbered list |
+| HeadingBlock | Text content to be formatted as a page header |
+| HeadingOneBlock | Largest sized heading |
+| HeadingTwoBlock | Second largest heading |
+| HeadingThreeBlock | Smallest heading |
+| BookmarkBlock | A link to another notion page |
+| FencedCodeBlock | A block made entirely of code |
+| QuoteBlock | Text content in the form of a quote |
+| FileBlock | Notion block of a file |
+| ImageBlock | A Notion block of an image |
+| TableBlock | A Notion block of a table |
+| RawBlock | A Notion block of raw text |
+| ToggleBlock | A Notion block for a toggled list |
+| CalloutBlock | A text block that is made to stand out from the rest |
+| NoopBlock | A none type Notion block |
+| TableOfContentsBlock | A Notion block that lists and links the headers of the page |
+| BreadcrumbBlock | A Notion block which denotes the current page location |
+| UnsupportedBlock | A Notion block that isn't supported by the program yet |
+| TemplateBlock | A template for a new page |
+| WarningBlock | A block that logs a warning for unsupported blocks |
+| ChildDatabaseBlock | A block that represents a database that is the child of the current page |
+| EmbedBlock | A block which embeds one of a selection of features into the page |
+| VideoBlock | A block that houses a video |
+| PdfBlock | A block that houses a PDF |
+| ChildrenPassThroughBlock | A block that passes along the children of it's parent |
+| ColumnBlock | A block that apears on the page as a column |
```

Comment:
The description of the ColumnBlock should explain how we convert the column blocks into markdown. In particular, we ignore the columns and just display the content in each column one after the other in the markdown, since there is no way to support columns in markdown. This is a good example of the non-obvious sort of content that should go into this table.